Kernel/Storage/SD/Registers.h

```diff
@@ -62,6 +144,46 @@ struct HostControlRegisterMap {
 };
 static_assert(AssertSize<HostControlRegisterMap, 256>());
 
+// SDHC Figure 1-10 : General Descriptor Table Format
+enum class DMAAction : u8 {
+    // ADMA 2
+    Nop = 0b000,
+    Rsv0 = 0b010,
+    Tran = 0b100,
+    Link = 0b110,
+    // ADMA 3
+    CommandDescriptor_SD = 0b001,
+    CommandDescriptor_UHS_II = 0b011,
+    Rsv1 = 0b101,
+    IntegratedDescriptor = 0b001,
```

Comment:
has the same bits as `CommandDescriptor_SD`, a mistake? (comment somehow if not?)

Reply:
Yes, should have been `0b111`